amplication/amplication
# Fix(amplication): align amplication services env variables to prod variables in all services

Close: #5038 

## PR Details

Align amplication services env variables to prod variables in all services.
With this change, now the local-dsg-runner is fully functional end-2-end from build > generate files > create pull request


## How to Test:
run the local dsg 
run amplication server
run amplication build manager
run amplication git pull request service
run amplication client

Click on the build button in the client on a service that is synced with GH.
The push to GH step should complete successfully

## PR Checklist

- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.
